app/person/tests.py

```diff
@@ -32,3 +39,23 @@ def test_name_label(self):
     person = Person.objects.get(id=1)
     label = person._meta.get_field('name').verbose_name
     self.assertEqual(label, 'Full name')
+  
+  def test_update_screens_button_renders(self):
+    User.objects.create_superuser(test_username, test_email, test_password)
+
+    c = Client()
+    c.login(username=test_username, password=test_password)
+    response = c.get('/admin/person/person/')
+  
+    self.assertEqual(response.status_code, 200)
+    self.assertTrue(update_screens_button_label in response.content.decode('utf-8'))
+
+  def test_click_update_screens_button_result(self):
+    User.objects.create_superuser(test_username, test_email, test_password)
+
+    c = Client()
+    c.login(username=test_username, password=test_password)
+    response = c.get('/admin/person/person/actions/UpdateScreens/', follow=True)
```

Comment:
Need to add `with disable_auto_indexing():` here.

Reply:
Thanks! This was added in f468a61.